site/profile/manifests/core/telegraf.pp

```diff
@@ -10,15 +10,14 @@
 # @param database
 #   The InfluxDB database for node metrics (as opposed to summit power metrics).
 #
-# @param host
+# @param influxdb_url
 #   The backing InfluxDB instance. At present the default DNS record is a CNAME.
 class profile::core::telegraf(
   String $password,
-  String $username = 'telegraf',
-  String $database = 'telegraf',
-  String $host     = 'metrics-ingress.ls.lsst.org',
+  String $username     = 'telegraf',
+  String $database     = 'telegraf',
```

Comment:
there's no concern on leaving the name of the db and the user in plain text?

Reply:
These are default values and don't really need to be secured; the password is safely stored and should provide sufficient security for our needs.